index.js

```diff
@@ -2,7 +2,7 @@
     A lot of the "appendChild" and "createElement" could be handled with string literals, 
     but this explains DOM manipulation a bit better than innerHTML.
 */
-
+// Nice. Would you say that this is safe for a production site?
```

Comment:
Are you talking about using es6 modules or the config file so easily seen? 
I'm assuming you are talking about the modules part and the explanation is philosophical in nature. 
Firstly for paying clients I would still be transpiling with with babel and webpack, especially if it is buisness facing mostly. Its just to protect the client from having to deal with the one high paying customer who insist on using Window XP still for some reason. =/. However I think it is close to production ready with windows 7 loosing support and edge, ff, and chrome all being evergreen in windows 8 (safari can still be problematic on older macs, but not as bad as IE).
For any project that is a project of my choice like a game, pet project,  or something else then yes I would use it in production. I started making websites in the days of IE 6 and it took developers consistently using the modern standards to get past that. So forward thinking and using modern standards are very important to me.